Introducing - 'add contact'

DanielMy777 opened the pull request:
# Description
A seeker can now send a connection request to an owner in order to start a connection process.
1) Adding the url needs to be inserted that leads to the 'add_new_contact' view.
2) Adding the 'add_new_contact' view that handles logic and errors of a seeker trying to add
    a new connection with an owner.
3) Adding two aiding test fixtures that log in as a seeker or owner.
4) Adding tests that test all cases from success to errors trying to use the 'add contact' feature.

## Manual Tests
Tried to manually force all errors and received a fitting feedback.
Tried to add a successful contact and received a fitting feedback.

## Additional Information
How to use:
While in a seeker profile, referring him to the url "/contacts/add/`<apartment_id>`" will 
add a new pending connection request between that seeker and owner of the apartment.

Will work more nicer after the merge of messages, but not needed.

### Issues closed by this PR
fix #113

nadavsu commented:
> While in a seeker profile, referring him to the url "/contacts/add/<apartment_id>" will
>add a new pending connection request between that seeker and owner of the apartment.

By <apartment_id>, you mean the owner ID right?

DanielMy777 commented:
> > While in a seeker profile, referring him to the url "/contacts/add/<apartment_id>" will
> > add a new pending connection request between that seeker and owner of the apartment.
> 
> By <apartment_id>, you mean the owner ID right?

@nadavsu An apartment ID, the PK of the 'Apartment' model, since a connection in made between a seeker and apartment models.

DanielMy777 committed:
Adding log in and log out fixtures

DanielMy777 committed:
Adding the 'add contact' feature

1) Addig a url request to add an owner as a connection.
2) Adding a view handeling logic & possibe errors of
   trying to add a new connection.
3) Adding tests to test multiple cases in the matter.

Micha546 reviewed (commented):
🔥

nadavsu reviewed (approved):
Looks good.

Micha546 reviewed (approved):
LGTM 🔥
